packages/core/src/state-management/repository.ts

```diff
@@ -819,6 +824,7 @@ export class Repository {
    * Adds the stream's RunningState to the in-memory cache and subscribes the Repository's global feed$ to receive changes emitted by that RunningState
```

Comment:
Haha I have no idea why this comment is here already but it seems to have perfectly predicted this future 😂